pkg/nfspv/nfsvolume.go

```diff
@@ -0,0 +1,340 @@
+/*
+Copyright © 2021 The MayaData Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package nfspv
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/mayadata-io/volume-events-exporter/pkg/collectorinterface"
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/strategicpatch"
+	"k8s.io/client-go/kubernetes"
+	corev1listers "k8s.io/client-go/listers/core/v1"
+)
+
+const (
+	OpenEBSCASLabelKey      = "openebs.io/cas-type"
```

Comment:
this is not specific to nfs PV. let's move it in pkg/controller

Reply:
Moved constant to pkg/controller/base_controller.go when the configuration is exploded then we will move to separate file or to openebs/api repository.